modin/core/execution/ray/implementations/cudf_on_ray/partitioning/partition.py

```diff
@@ -201,15 +201,15 @@ def width(self):
             return self._width_cache
         return self.gpu_manager.width.remote(self.get_key())
 
-    def mask(self, row_indices, col_indices):
+    def mask(self, row_labels, col_labels):
```

Comment:
Similar changes should be added to all files regarding partitions.

Reply:
Agree, please add type hints to these places as well.

Reply:
should these args have type hints? presumably Optional[List[Hashable]] = None